docs/Installation/Installation.md

```diff
@@ -4,6 +4,10 @@ See LICENSE.txt for details.
 \endcond
 # Installation {#installation}
 
+This page details the installation procedure for SpECTRE on personal computers. For instructions on installing SpECTRE on clusters please refer to the following page:
```

Comment:
Good thing to add :)

- 80 char
- just put the link on the same line, e.g. `refer to \ref ...`